saleor/product/migrations/0112_auto_20191209_0437.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.7 on 2019-12-09 10:37
```

Comment:
If you move warehouse and stock logic from 0111 to warehouse.init_migration and stock.Init_migration, then you should be able to easily create one migration for product(instead of 111 and 112)